README.md

```diff
@@ -316,7 +325,7 @@ fig.savefig("docs/example_multiple_lines.png")
 
 ![Recall with multiple lines](docs/example_multiple_lines.png)
 
-## Metrics
+## Statistical results
```

Comment:
I'm struggling with the wording here. The API uses the word "metrics". I decided to use metrics internally because we are talking about metrics and scores, not statistics. This keeps the API and wording in line with the wording used machine learning packages like sklearn and tensorflow (we depend on both packages). 

However, the CLI uses "stats", which is legacy. I was thinking about renaming this. This is actually the best moment for renaming. I don't see any stats we want to include here in the nearby future, as dataset statistics are moved to datatools extension.